src/Microsoft.DotNet.Wpf/test/MultiTargeting.props

```diff
@@ -4,28 +4,10 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <DefineConstants Condition="$(TargetFramework.StartsWith('net4'))">$(DefineConstants);NET4X</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.StartsWith('netcoreapp3'))">$(DefineConstants);NETCOREAPP3_X</DefineConstants>
-
-    <DefineConstants Condition="$(TargetFramework.Equals('net40'))">$(DefineConstants);NET40</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net45'))">$(DefineConstants);NET45</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net451'))">$(DefineConstants);NET451</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net452'))">$(DefineConstants);NET452</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net453'))">$(DefineConstants);NET453</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net46'))">$(DefineConstants);NET46</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net461'))">$(DefineConstants);NET461</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net462'))">$(DefineConstants);NET462</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net47'))">$(DefineConstants);NET47</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net471'))">$(DefineConstants);NET471</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('net472'))">$(DefineConstants);NET472</DefineConstants>
-
-    <DefineConstants Condition="$(TargetFramework.Equals('netcoreapp3.0'))">$(DefineConstants);NETCOREAPP3_0</DefineConstants>
-    <DefineConstants Condition="$(TargetFramework.Equals('netcoreapp3.1'))">$(DefineConstants);NETCOREAPP3_1</DefineConstants>
     <DefineConstants Condition="$(TargetFramework.Equals('net6.0'))">$(DefineConstants);NETCOREAPP5_0</DefineConstants>
```

Comment:
This one looks odd...

Reply:
I do not have whole visibility for the project so was not sure about this file. Also changing define may touch a lot of files.

Reply:
This was a general observation, more aimed at the WPF team.